spring-cloud-kubernetes-integration-tests/spring-cloud-kubernetes-client-config-it/pom.xml

```diff
@@ -60,6 +77,56 @@
 				<filtering>true</filtering>
 			</resource>
 		</resources>
+
+		<plugins>
+			<!-- build image in the 'package' phase, and ignore plain tests -->
```

Comment:
@ryanjbaxter every integration test has these 3 parts: 

- build the image in the `package` phase
- skip plain tests with `skipTests` in `maven-surefire-plugin`
- run integration tests in the `maven-failsafe-plugin`

In this manner, when running `mvn clean install`, we will first build the image and then run the tests